config/routes.rb

```diff
@@ -142,10 +142,11 @@
       get   "/settings" => "communities#settings",        as: :settings
       patch "/settings" => "communities#update_settings", as: :update_settings
 
+      get "getting_started_guide(/*all)" => "getting_started_guide#index", as: :getting_started_guide
```

Comment:
Seems that the route name `:getting_started_guide` is now clashing with the one route in the styleguide. It's no not clear which route will `getting_started_guide_path` URL helper generate. Maybe the styleguide name could be renamed to something else, like `getting_started_guide_styleguide` etc.?